apps/nextjs-example/package.json

```diff
@@ -17,6 +17,7 @@
     "@aptos-labs/wallet-adapter-mui-design": "workspace:*",
     "@aptos-labs/wallet-adapter-react": "workspace:*",
     "@aptos-labs/wallet-standard": "^0.0.11",
+    "@atomrigslab/aptos-wallet-adapter": "^0.1.5",
```

Comment:
this should live in the core package, I can help with that

Reply:
> this should live in the core package, I can help with that

Thank you for the guide and help.

Should we also install our adapter's core dependency and set it up like in [this one](https://github.com/aptos-labs/aptos-wallet-adapter/blob/sdk_standard_wallets/packages/wallet-adapter-core/src/AIP62StandardWallets/sdkWallets.ts)?

Our adapter includes both DevTWallet and ProdTWallet. I intended to allow dApp developers to choose which one to use during development and production. While installing the dependency in the adapter is okay, enabling initialization might be better, as it lets dApps decide which wallet to use.

Reply:
> > this should live in the core package, I can help with that
> 
> Thank you for the guide and help.
> 
> Should we install TWallet adapter in the core library dependency and set it up like in [this one](https://github.com/aptos-labs/aptos-wallet-adapter/blob/sdk_standard_wallets/packages/wallet-adapter-core/src/AIP62StandardWallets/sdkWallets.ts)?
> 
> Our adapter includes both DevTWallet and ProdTWallet. I intended to allow dApp developers to choose which one to use during development and production. While installing the dependency in the adapter is okay, enabling dapp to register TWallet might be better, as it lets dApps decide which one to use.

SDK wallets should be installed and managed in the adapter core package.

That way, a wallet benefits from visibility, custom UI, adapter extra features and logic while the dapp utilizes the adapter for more security, state management and features.

Also, having a dapp uses the wallet adapter and a separate sdk wallet npm package can create state management risks and a buggy UI and the wallet/dapp will not utilize the adapter for all the extra features and logic.

Yes the process of adding a SDK wallet should look similar to [that](https://github.com/aptos-labs/aptos-wallet-adapter/blob/sdk_standard_wallets/packages/wallet-adapter-core/src/AIP62StandardWallets/sdkWallets.ts), tho need some more configuration - I can help with that when I am back next week.